core/conn/jdbc_type2/native/Debug.cpp

```diff
@@ -183,8 +184,9 @@ static const char *UCase(const char *text)
 
 static unsigned long GetThreadId(void)
 {
-	_TSLX_t thread_id = tslx_ext_pthread_self();
-	return((unsigned long) thread_id.field1);
+	//_TSLX_t thread_id = tslx_ext_pthread_self();
+	//return((unsigned long) thread_id.field1);
+        return pthread_self();
```

Comment:
I will use syscall(SYS_gettid).
 For removing TSLX thread references, I think it's better to file a new issue to do it.